estate/models/__init__.py

```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
This header is not needed anymore. It should not be added to newly created python file in the Odoo codebase.